addons/flag-menu/context_menu.css

```diff
@@ -0,0 +1,42 @@
+/* Hide context menu items related to other addons by default */
+#sa-flag-menu-fps,
+#sa-flag-menu-mute {
+  display: none;
+}
+
+/* Styles reused from https://github.com/scratchfoundation/scratch-gui/blob/develop/src/components/context-menu/context-menu.css */
```

Comment:
These styles will soon be outdated because Scratch's [React 18 PR](https://github.com/scratchfoundation/scratch-editor/pull/299) changes context menus. Can you use `addon.tab.scratchClass` so that the addon doesn't need to duplicate Scratch CSS?

Reply:
Alright. It seems to work fine.